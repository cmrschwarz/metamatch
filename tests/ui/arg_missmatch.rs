use metamatch::metamatch;

fn main() {
    let x = 42;
    metamatch! {
        match f {
            #[expand((A, B) in [(1, 2), (3)])]
            A => A + B,
            _ => (),
        }
    }
}
