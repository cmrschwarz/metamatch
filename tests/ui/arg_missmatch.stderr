error: tuple pattern does not match `int`
 --> tests/ui/arg_missmatch.rs:7:26
  |
7 |             #[expand(for (A, B) in [(1, 2), (3)])]
  |                          ^^^^^^
